recipe/meta.yaml

```diff
@@ -28,8 +28,12 @@ requirements:
     - numpy >=1.12.1
     - protobuf >=3.4.0
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
     - backports.weakref >=1.0rc1
+    - astor
+    - grpcio
+    - termcolor
+    - gast
+    - tensorboard >=1.7.0,<1.8.0
```

Comment:
Is this valid? `tensorflow` clearly pins `tensorboard` to `>= 1.8.0, < 1.9.0` so you'd assume there's a good reason for that?
https://github.com/tensorflow/tensorflow/blob/v1.8.0/tensorflow/tools/pip_package/setup.py#L34-L43

I'd be inclined to respect the pinning as per their `setup.py` and just update `tensorboard`